### PR TITLE
Fix UI bugs

### PR DESCRIPTION
**Pull request by Nazeeh2000**
Add dark and light mode for CountryInfo tab, Fix countrySearch and Infinite state updates in App.js.

### CODE REVIEW

**Comment by JFQueralt** [comment]
@Nazeeh2000 Please check the details on the checks. They seem to be rather straightforward (same as prior PRs).

File: MAP/src/components/LanguageSelector/LanguageSelector.js [hunk 1]
```diff
@@ -34,7 +34,9 @@ const LanguageSelector = ({ dark }) => {
               className={`${classes.LangOpt} ${
                 language === selectedLang && classes.active
               } ${dark && classes.dark}`}
-              onClick={() => setSelectedLang(language.replace(/\s+/g, '-'))}
+              onClick={() => {
+                // languageChangeHandler(language.replace(/\s+/g, '-'))
```
[the remark this answers is not in the document]
Comment (Nazeeh2000) [reply MAP/src/components/LanguageSelector/LanguageSelector.js:38]: Okay I'll do the same :)

**Comment by jeff-knurek** [comment]
I noticed that `node_modules/` directory is being added in the PR at the top level of the repo

@Nazeeh2000 @nakullondhe please remove this code from the PR - maybe also include a `.gitignore` at the top level so that doesn't happen again _(even though it shouldn't be possible)_

**Comment by jeff-knurek** [comment]
@stanley-miao it looks like there's still a lot of commented out "dead code" to clean up in this PR

**Comment by stanley-miao** [comment]
> @stanley-miao it looks like there's still a lot of commented out "dead code" to clean up in this PR

@jeff-knurek should I just delete all of the commented code? I'm not sure if any of it is of importance.

**Comment by jeff-knurek** [comment]
> should I just delete all of the commented code? I'm not sure if any of it is of importance.

yeah, the great thing about version control (git), is that the history still exists. so if at some point someone wants to go back to the code they can

**Review by jeff-knurek — APPROVED** [review]
LGTM - still need to iterate on other fixes, but a good first step

**Pull request merged**